alphagov/paas-admin
# [#162443556] Change method of obtaining user spaces

What
----

Rather than relying on Cloud Foundry's API to provide all the spaces a user has permissions for, this determines them programmatically.

1 Get all spaces in the org
2 Get all users for those spaces
3 Build an object, keyed by user, with roles and spaces

Also, updated the template as the object's structure has changed.

How to review
-------------
1. Code Review
2. Deploy and test:
   1. Deploy to a dev environment, and route `admin.` to it
   2. Log in as an Organisation manager
   3. Confirm you can see Spaces for each user in `/organisations/org_guid/users`

Who can review
---------------

NOT @tnwhitwell, @richardTowers
